src/main/scala/com/ubirch/filter/services/kafka/DefaultFilterService.scala

```diff
@@ -273,7 +270,7 @@ abstract class AbstractFilterService(cache: Cache, finder: Finder, config: Confi
     val result = send(data.cr.toProducerRecord(topic = producerForwardTopic))
```

Comment:
in this `forwardUPP` method, try-catch can be removed as well because the `set` method returns `Future` now.

Reply:
I just changed them to be a recover instead, as they seemed to be implemented intentionally. Or did you have something else in mind?